src/azure-cli/azure/cli/command_modules/acs/azurecontainerstorage/_helpers.py

```diff
@@ -276,6 +303,7 @@ def get_current_resource_value_args(
 def get_desired_resource_value_args(
```

Comment:
This is different from the impletion in aks-preview, missing `perf_tier_updated`, expected?

Reply:
Resolved